### DISCUSSION TITLE
All my pages in pagespeed insights showing this problem

### DISCUSSION THREAD

**Discussion by rahul3103** [vercel/next.js#16960] [Help]
```If you are server-side rendering any React components, consider using `renderToNodeStream()` or `renderToStaticNodeStream()` to allow the client to receive and hydrate different parts of the markup instead of all at once.```

Even the framework also its showing taking so much time. How can I solve this issue, or how should I proceed ahead?

<img width="737" alt="Screenshot 2020-09-09 at 5 59 12 PM" src="https://user-images.githubusercontent.com/10813252/92598251-2ff12280-f2c6-11ea-86a1-f05d590d90e0.png">

lighthouse pagespeedinsights

**Comment by vasconx** [comment]
Yessss me too!

**Reply by oknoorap** [reply]
yes, please how to fix this

**Comment by M1ck0** [comment]
Are you testing this in production or development mode?

**Reply by rahul3103** [reply]
production

**Comment by redian** [comment]
There is an open discussion about this topic over here: https://github.com/vercel/next.js/issues/1209
